packages/remirror__extension-callout/__stories__/callout.stories.tsx

```diff
@@ -2,74 +2,138 @@ import 'remirror/styles/all.css';
 
 import { EmojiButton } from '@joeattardi/emoji-button';
 import { Blobmoji } from '@svgmoji/blob';
-import React, { useCallback } from 'react';
+import React, { useCallback, useEffect, useRef } from 'react';
 import { CalloutExtension } from 'remirror/extensions';
 import svgmojiData from 'svgmoji/emoji.json';
-import { EditorView, htmlToProsemirrorNode, ProsemirrorNode } from '@remirror/core';
+import { htmlToProsemirrorNode, ProsemirrorNode } from '@remirror/core';
 import { ProsemirrorDevTools } from '@remirror/dev';
-import { Remirror, ThemeProvider, useRemirror } from '@remirror/react';
+import {
+  Remirror,
+  ThemeProvider,
+  useCommands,
+  useRemirror,
+  useRemirrorContext,
+} from '@remirror/react';
 
 export default { title: 'Callouts' };
 
-const renderDialogEmoji = (node: ProsemirrorNode, view: EditorView, getPos: () => number) => {
-  const { emoji: prevEmoji, type } = node.attrs;
-  const emoji = document.createElement('span');
-  emoji.textContent = prevEmoji;
+const EmojiPicker = () => {
+  const pickerRef = useRef(new EmojiButton({ position: 'bottom', autoFocusSearch: false }));
+  const { updateCallout } = useCommands();
+  const { view } = useRemirrorContext();
+  const pos = useRef(-1);
 
-  const picker = new EmojiButton({
-    position: 'bottom',
-    autoFocusSearch: false,
-  });
+  const handleClickEmoji = useCallback(
+    (e: MouseEvent) => {
+      const target = e.target as HTMLElement;
+
+      e.preventDefault();
+
+      if (!target.matches('[data-emoji-container]')) {
+        return;
+      }
+
+      /**
+       * Find the document position of the click element.
+       */
+      pos.current = view.posAtDOM(target, 0);
+
+      pickerRef.current.togglePicker(target);
+    },
+    [view],
```

Comment:
Not a biggie, but I think you could use `[view.posAtDOM]` here?

Reply:
The`exhaustive-deps` eslint rule prevents me from doing it, and if I assign `posAtDOM` to a new variable, it would cause a change to `this`, and making things a bit more tricky.
Do you have any idea to solve the problem?

Reply:
I suppose this is only Storybook, so not a big deal - I just figured `view` is like to change a lot, although I guess as React compares with `Object.is` this is probably fine.